packages/widget/src/components/MainButton.tsx

```diff
@@ -113,12 +113,11 @@ export const LoadingButton = ({
 const MainButtonText = styled(Text).attrs({
   fontWeight: "bold",
   capitalize: true,
-  letterSpacing: "-0.015em",
 })`
-  font-size: 24px;
+  font-size: 18px;
```

Comment:
Idk about simply reducing the font size across the board, and it differs from the design

Reply:
won't this look small for the usual "swap" button, "connect wallet" and such?

Reply:
+1 on this

Reply:
can we make this font size only reduce to 18px if there's a certain amount of text/characters?

Reply:
Im playing around with an AutoResizeText component but its really ugly with styled componenets